utils/zerovec/src/ule/plain.rs

```diff
@@ -25,6 +25,18 @@ macro_rules! impl_byte_slice_size {
             pub fn as_bytes(&self) -> &[u8] {
                 &self.0
             }
+
```

Comment:
Nit: Move this function down to be next to `from_byte_slice_unchecked_mut`. That function also looks a bit fishy and maybe we should clean it up.